src/autoLaunch.js

```diff
@@ -0,0 +1,7 @@
+import AutoLaunch from 'auto-launch';
+import os from 'os';
+
+
+export function autoStartOnBoot() {
+  console.log('hiiiiiiii');
```

Comment:
Lol, while I enjoy the friendly demeanor, we should probably log something more useful here.
e.g.
`console.log('Auto starting on boot...');`

Reply:
settingsHelper.js

`function whichSetting(settingsBool, configSetting) {
    if (settingsBool == configSetting) {
      console.log('autolaunch setting already configured');
    } else if (configSetting) {
      console.log('autolaunch enabled');
      kioskAutoLaunch.enable();
    } else {
      console.log('autolaunch disabled');
      kioskAutoLaunch.disable();
    }
  }`